Makefile

```diff
@@ -95,12 +95,8 @@ help: ## Display this help.
 
 ##@ Development
 
-.PHONY: manifests
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
```

Comment:
@kfswain @nirrozenbaum Do you need anything beyond CRDs that could have been generated here? (I'm assuming not, certainly not a webhook, and probably not RBAC, but just want to make sure I'm not missing some EPP dependencies here).